client/asset/btc/rpcclient.go

```diff
@@ -768,8 +769,26 @@ func (wc *rpcClient) walletLock() error {
 }
 
 // sendToAddress sends the amount to the address. feeRate is in units of
-// sats/byte.
+// sats/byte. If there is not a fee rate positional param, it is used
+// legacySendToAddress instead.
 func (wc *rpcClient) sendToAddress(address string, value, feeRate uint64, subtract bool) (*chainhash.Hash, error) {
+	// 1e-5 = 1e-8 for satoshis * 1000 for kB.
+	var txid string
```

Comment:
nit: Can move this down and not declare until after the possible return.

Reply:
I still think this declaration should be moved after the conditional.  It looks sloppy up here.

Reply:
oh sorry, skiped this comment. mb